package.json

```diff
@@ -0,0 +1,27 @@
+{
+	"name": "express_app",
+	"version": "1.0.0",
+	"description": "",
+	"main": "server.ts",
+	"scripts": {
+		"start": "ts-node server.ts",
+		"start:dev": "nodemon --exec ts-node server.ts",
+		"build": "tsc -p ."
+	},
+	"author": "Alla Oleksyn",
+	"license": "ISC",
+	"devDependencies": {
+		"@types/config": "^3.3.1",
+		"@types/express": "^4.17.18",
+		"@types/node": "^20.8.2",
+		"@types/swagger-jsdoc": "^6.0.1",
+		"@types/swagger-ui-express": "^4.1.4",
+		"nodemon": "^3.0.1",
+		"swagger-jsdoc": "^6.2.8",
+		"swagger-ui-express": "^5.0.0",
+		"typescript": "^5.2.2"
+	},
```

Comment:
You try to run `ts-node` in `start:dev` script but do not have proper dev dependency in the list. If it works on your local machine, probably, this NPM package installed globally, but better to have all related dependencies in project's package.json list.

Reply:
You are right, thanks. Added ts-node to devDependencies.